testsuite/unittests/test_bernoulli_synapse.sli

```diff
@@ -0,0 +1,103 @@
+/*
+ *  test_bernoulli_synapse.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/* BeginDocumentation
+    Name: testsuite::test_bernoulli_synapse - test of bernoulli_synapse connection
+
+    Synopsis: (test_bernoulli_synapse) run -> measure average number of spikes transmitted 
+    by a Bernoulli synapse (static synapse with stochastic transmission)
+
+    Description:
+    1000 spikes generated by spike_generator are sent to a parrot neuron using a 
+    bernoulli_synapse with transmission probability 25%. 
+    If the average number of spikes detected by spike detectors is within three standard 
+    deviations from the mean of the binomial distribution, the synapse works fine.
+    Also tests the interval of possible probabilities, [0, 1].
+
+    FirstVersion: June 2017
+    Author: Maximilian Schmidt, Milena Menezes Carvalho
+*/
+
+(unittest) run
+/unittest using
+
+% test parameters -------------------------------------------------
+/N_spikes 1000 def                                    % number of spikes send via the bernoulli synapse to each parrot neuron
+/p 0.25 def                                           % transmission probability of the bernoulli synapse
+/margin N_spikes p mul 1 p sub mul sqrt 3 mul def     % allowed absolute deviation: three standard deviations of the binomial distribution with (N_spikes, p)
```

Comment:
Move this comment above the line it documents, it is too long to be stretch out to the right.